source/server/server.cc

```diff
@@ -410,6 +410,26 @@ void InstanceImpl::initialize(const Options& options,
   // instantiated (which in turn relies on runtime...).
   Runtime::LoaderSingleton::get().initialize(clusterManager());
 
+  Runtime::LoaderSingleton::get().startRtdsSubscriptions([this]() { onRuntimeReady(); });
```

Comment:
What happens if there is not RTDS? I'm expecting it calls back gracefully, maybe worth a comment.

Reply:
done